apps/evans/components/pages/home/HeroBlock.tsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import cx from 'classnames';
+import Head from 'next/head';
 import Dual from '@/components/core/Dual';
-import BackgroundImage from '@/components/core/BackgroundImage';
-import TreeRootsImage from '@/public/images/tree-roots.webp';
+import TreeRootsLg from '@/public/images/tree-roots.webp';
+import TreeRootsSm from '@/public/images/tree-roots.small.webp';
```

Comment:
the tree roots image at the top of the homepage, was the main factor in lighthouse LCP, so i hand-optimized it. in the process, figured out some cool stuff about preloading and how next image works that i'll share with you when I'm back from CR.